jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -0,0 +1,115 @@
+import functools
+import numpy as np
+import tensorflow as tf # type: ignore[import]
```

Comment:
Would it be possible to remove the dependency on tf for this module? It is not important, but I feel that we are adding such a big dependency for so little benefit.

Reply:
Done in `7da907f`, but the alternative looks a little bit hacky. OK with it if you are and the cost of importing TF justifies it (I envision that this function is probably the only reason we'd need TF there)